examples/src/main/scala/za/co/absa/atum/utils/SparkJobRunnerMethods.scala

```diff
@@ -30,7 +30,7 @@ trait SparkJobRunnerMethods {
     val jobClassSymbol = universe runtimeMirror jobClass.getClassLoader classSymbol jobClass
     val jobInstance =
       if (jobClassSymbol.isModuleClass) jobClass getField "MODULE$" get jobClass
-      else jobClass.newInstance
+      else jobClass.getDeclaredConstructor().newInstance()
```

Comment:
I don't think this was actually the reason for the _Warning_. I believe the line 55 is. And to get rid of that _feature_ warning add:
`import scala.language.reflectiveCalls`